utopia-remix/app/handlers/validators.ts

```diff
@@ -10,16 +10,20 @@ export function validateProjectAccess(
   {
     errorMessage,
     status,
+    idRouteParam = 'id',
```

Comment:
please let's not give an arbitrary default here and instead make this explicit (sure sure it'll need updating across the board but it's good)

Reply:
fixed by passing a `getProjectId` explicit function